src/main/java/controller/LottoGame.java

```diff
@@ -0,0 +1,67 @@
+package controller;
+
+import java.util.Objects;
+
+import model.Lotto;
+import model.LottoNumberChoiceRandom;
+import model.LottoPurchaseCount;
+import model.Lottos;
+import model.RewardCalculator;
+import view.InputView;
+import view.ResultView;
+
+public class LottoGame {
+	InputView inputView = new InputView();
+	ResultView resultView = new ResultView();
```

Comment:
```suggestion
	private final InputView inputView = new InputView();
	private final ResultView resultView = new ResultView();
```

접근제어자를 붙여주는 습관을 들여보아요!